From 50787987c730042e3cb6c12b6b65f6ffa67a1209 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sun, 28 Apr 2024 15:01:12 +0300
Subject: [PATCH] [SW-15774] scripts: dirs: change to dev_prefix

HL_DEV may be <board> or <board>_arc. DEV_PREFIX will be <board>
for both cases. Therefore ${HL_DEV}_arc can be extract to
<board>_arc_arc.

Change-Id: Ic3fc4062c9fa8655f04ad1330512244910dd2f0a
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/board/habanalabs/common/dirs.sh b/board/habanalabs/common/dirs.sh
index bad3f9c..12cc9a0 100644
--- a/board/habanalabs/common/dirs.sh
+++ b/board/habanalabs/common/dirs.sh
@@ -1,7 +1,7 @@
 # NOTE: should be in sync with scripts/make/common/hl_dirs.mk
 
 export DEV_BASE_DIR=${BR2_EXTERNAL}/output/${HL_DEV}/
-export DEV_PREFIX_BASE_DIR=${BR2_EXTERNAL}/output/${DEV_PREFIX}/
+export DEV_PREFIX_BASE_DIR=${BR2_EXTERNAL}/output/${DEV_PREFIX}_arc/
 export IMGS_DIR=${DEV_BASE_DIR}/images/
 export IMGS_PROD_DIR=${DEV_PREFIX_BASE_DIR}/images/
 export STAGE_DIR=${IMGS_DIR}
